packages/lodestar-utils/src/logger/format.ts

```diff
@@ -23,7 +24,12 @@ export const defaultLogFormat = format.combine(
 export function serializeContext(context?: Context | Error): string {
   if (!context) return "";
   if (context instanceof Error) {
-    return "Error: " + context.message + "\n" + context.stack;
+    const errObj = toJson(context);
+    if (errObj != null && typeof errObj === "object" && !Array.isArray(errObj)) {
+      delete errObj["message"];
```

Comment:
Why aren't we printing error message?

We still have and will have bunch or errors originating from libraries and stuff that won't have meta so we would end-up being clueless about error.

Reply:
Agree this should work with Errors that we don't control

Reply:
This should work well for errors we don't control.
The `error.message` is part of the `error.stack` :o)

Eg:
```typescript
logger = new WinstonLogger();
logger.info("A terrific error", new Error("this is an error"));
```
will output
```
2020-09-30 13:38:59  []                 info: A terrific error
Error: this is an error
    at repl:1:33
    at Script.runInThisContext (vm.js:120:18)
    at REPLServer.defaultEval (repl.js:433:29)
    at bound (domain.js:427:14)
    at REPLServer.runBound [as eval] (domain.js:440:12)
    at REPLServer.onLine (repl.js:760:10)
    at REPLServer.emit (events.js:327:22)
    at REPLServer.EventEmitter.emit (domain.js:483:12)
    at REPLServer.Interface._onLine (readline.js:329:10)
    at REPLServer.Interface._line (readline.js:658:8)
```

Reply:
True, tho it is possible for some insane dev to throw Errors that do not have the stack property. Maybe we can log the message only if there is no stack.